omod/src/test/java/org/openmrs/module/idgen/web/resource/LogEntryResourceTest.java

```diff
@@ -0,0 +1,32 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.idgen.web.resource;
+
+import org.openmrs.module.idgen.LogEntry;
+import org.openmrs.module.idgen.rest.resource.LogEntryResource;
+import org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResourceTest;
+
+public class LogEntryResourceTest extends BaseDelegatingResourceTest<LogEntryResource, LogEntry> {
+
+	@Override
+	public String getDisplayProperty() {
+		return null;
+	}
```

Comment:
Can you look at the resource test i pointed you in the ticket description and then put some thing in these method bodies? You can even just copy some ideas from the pull request i pointed you to done by one of your team members.

Reply:
Dont know whether I am the one getting this wrong but the log entries seem to not have universally unique identifiers. How do I go about the getUuidProperty and newObject methods

Reply:
You are correct on that. Just use the id as uuid

Reply:
what do I do for the newObject method then. If I use the getLogEntries, it takes the arguments (IdentifierSource source, Date fromDate, Date toDate, String identifier, User generatedBy, String comment) so I get the "not applicable for the arguments (String)" error

Reply:
Have you looked at how the other pull request deals with the newObject() method?

Reply:
Came across [the Module Resource Test](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/ModuleResource1_8Test.java) which I've used to guide me.

Reply:
That was so clever of you! 👍

Reply:
Thanks. The errors in the tests are still persistent though.

- > getUserService()  &  getService() are undefined for the LogEntryResourceTest 

_(but  I've used them in other classes without any issues)_
- > Failed to load ApplicationContext

_for all other tests_

Reply:
Commit them and i will take a look to advise.

Reply:
I've committed all I've done so far